packages/malloy-interfaces/thrift/malloy.thrift

```diff
@@ -453,11 +408,13 @@ struct NumberCell {
 struct NullCell {}
 
 struct TimestampCell {
-  1: required string timestamp_value, // TODO another way to represent dates?
+  // TODO another way to represent dates?
+  1: required string timestamp_value,
 }
 
 struct DateCell {
-  1: required string date_value, // TODO another way to represent dates?
+  // TODO another way to represent dates?
+  1: required string date_value,
 }
 
 struct JSONCell {
```

Comment:
Currently there is a single `Result` struct that is used for both a "result with data" and a "result with just SQL". Should these be separated? (They are the same now because the _only_ difference between the two is the presence of the `data` field)

Reply:
MToy likes something like this better (but doesn't feel too strongly):

```
struct CompileResult {
  2: required Schema schema,
  3: optional string sql,
  4: required string connection_name,
  5: optional list<Annotation> annotations,
  6: optional list<Annotation> model_annotations,
  7: optional string query_timezone,
  8: optional list<Annotation> source_annotations,
}

struct RunResult {
  1: required Data data,
  2: required CompileResult compile_result,
}
```